include/SFML/Window/Event.hpp

```diff
@@ -87,6 +87,16 @@ class Event
         int y; ///< Y position of the mouse pointer, relative to the top of the owner window
     };
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Mouse move raw event parameter (MouseMovedRaw)
```

Comment:
Parameters not parameter.

Reply:
I never worked with this feature. Is it ok if i just add a commit to my pull request and click "Resolve conversation" on the respective suggestion?

Reply:
Usually yes, that's actually the purpose of that feature.